BUILD

```diff
@@ -34,14 +34,30 @@ load("@vaticle_dependencies//distribution:deployment.bzl", "deployment")
 load(":deployment.bzl", github_deployment = "deployment")
 
 
+genrule(
+    name = "python-ffi",
+    outs = ["typedb/typedb_client_python.py"],
+    srcs = ["@vaticle_typedb_driver_java//rust:typedb_client_python"],
+    cmd = "cp $< $@",
+    visibility = ["//visibility:public"]
+)
+
+genrule(
+    name = "python-lib",
```

Comment:
`native_client_binary`

Reply:
Files and srcs names should be:
`native_client_binary.so`
and
`//rust:_typedb_client_python_binary`

Reply:
Renamed, but not exactly to these names.